### DISCUSSION TITLE
How to get all tables in an ocr pdf as attached, and then parse all cells to get coordinates, then use the coordinates to re-ocr with another engine, then update the ocr result in pdf?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#3351] [General]
[table2TextOverImage.pdf](https://github.com/pymupdf/PyMuPDF/files/14892076/table2TextOverImage.pdf)

How to get all tables in an ocr pdf as attached, and then parse all cells to get coordinates, then use the coordinates to re-ocr with another engine, then update the ocr result in pdf?

**Comment by JorjMcKie** [comment]
Use `tabs = page.find_tables()`, then `tab = tabs[0]` to access the table itself.
The list tab.cells contains a rect-like tuple for each table cell.
I cannot answer the remaining part of the question.
